x/examples/outline-pwa/README.md

```diff
@@ -1,19 +1,345 @@
+    }
+    ```
+
+  * Proxy all app requests after the proxy is initialized using `ProxyController`:
+
+    ```kotlin
+    // NOTE: This affects all requests in the application
+    ProxyController.getInstance()
+        .setProxyOverride(
+            ProxyConfig.Builder()
+                .addProxyRule(this.proxy!!.address())
+                .build(),
+            {
+                runOnUiThread {
+                    // Capacitor does not expose a way to defer the loading of the webview,
+                    // so we simply refresh the page
+                    this.bridge.webView.reload()
+                }
+            },
+            {}
+        )
+    ```
+
+  * Turn off the proxy in `onDestroy`:
+
+    ```kotlin
+    override fun onDestroy() {
+        this.proxy?.stop(3000)
+        this.proxy = null
+
+        super.onDestroy()
+    }
+    ```
+
+## Verify with Packet Tracing
+
+* **iOS:** Load XCode and connect your iOS device. Make sure that your device allows your developer certificate (Check `Settings > General > VPN & Device Management` on your iOS device). Go to `Product > Profile` in XCode and select the "Network" option from the list of presets. Pressing the record button on the window that pops up should launch your app and record its traffic. Once done, set the view on the packet trace data to `History` and among the results you should see the DNS address `dn9.quad9.net`. 
+* **Android:** Start the emulator with `npx cap run android`. Use Wireshark to capture network traffic. Filter by `ip.addr == 9.9.9.9` (your chosen DNS server). You should see TCP and TLS traffic, indicating that your app is using DNS over HTTPS (DoH).
+
+## Building and Distributing your App
+
+### iOS
+
+1. Create a developer account and add it to XCode Settings (`XCode > Settings... > Accounts`)
+2. Select your App in the file explorer and pick your Account's Team in `Signing & Capabilities`
+3. Connect your iOS device and select it as your build target in the top bar. XCode should automatically create a provisioning profile for the device and app combination.
+4. Build a production archive with `Product > Archive`
+5. On success, a window should pop up with your app listed in it. Select `Distribute App` and follow the instructions.
+
+### Android
+
+First, generate a Key Store and use it to sign your app with Android Studio - follow these instructions: https://developer.android.com/studio/publish/app-signing#generate-key
+
+Note that you can choose to release your app as either an android app bundle (`.aab`) or an APK (`.apk`).
+1. You need an android app bundle (`.aab`) to release your app in the Google Play Store. For this you will have to have a [Google Play Developer Account](https://play.google.com/console/u/0/developers) and at least twenty trusted testers to unlock production access.
+2. APKs (`.apk`) can be freely sideloaded onto your user's devices. For an APK, you will have to take care of distribution yourself.
+
+**Important Notes:**
```

Comment:
This section feels like it should be at the top of the README, not at the bottom.